include/fmt/core.h

```diff
@@ -96,6 +96,20 @@
 #  define FMT_CONSTEXPR_DECL
 #endif
 
+#if __cplusplus >= 201402L
+#  define FMT_CONSTEXPR14 constexpr
+#else
+#  define FMT_CONSTEXPR14
+#endif
+
+#if __cplusplus >= 202002L
+#  define FMT_CONSTEXPR20 constexpr
+#  define FMT_IS_CONSTANT_EVALUATED std::is_constant_evaluated()
```

Comment:
This should be a function rather than a macro.

Reply:
Done.
I think macro was better here because it's just way simpler than a function.